subprojects/packagefiles/yyjson/meson.build

```diff
@@ -0,0 +1,42 @@
+project(
+  'yyjson',
+  'c',
+  version: '0.12.0',
+  license: 'MIT',
+  meson_version: '>=1.1.0',
```

Comment:
1.1 is only needed for the name `meson.options`, which isn't really worth the tighter compatibility constraints.  Better to rename to `meson_options.txt` and drop this to `0.54.0`.